src/components/CommentListing.jsx

```diff
@@ -0,0 +1,10 @@
+const CommentListing = (comment) => {
+  return (
+    <article className="comment-listing">
+      <div className="comment-author">{comment.author}</div>
+      <p className="comment-body">{comment.body}</p>
```

Comment:
Is this all the info you want to show for each comment? Fine if so but thought I would ask

Reply:
The main info I would say is missing (for now anyway) is a time of posting the comment. However, I left it out as I woud rather not display that as a timestamp. Timestamps look ugly in my opinion and not very user friendly. I would like to prioritise implementing further functionality first and return to it as I was considering looking into working out how to display the time passed since the comment has been posted rather than the actual time of the posting.